.github/workflows/stale.yaml

```diff
@@ -0,0 +1,24 @@
+name: Close stale PRs
+
+on:
+  schedule:
+  - cron: "00 0 * * *" # runs at 00:00 daily
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v5.1.1
+      name: Clean up stale PRs
+      with:
+        repo-token: ${{ secrets.stale_bot }}
+        stale-pr-message: "👋 This pull request has been marked as stale because it has been open with no activity. You can: comment on the issue or remove the stale label to hold stale off for a while, add the `Keep` label to hold stale off permanently, or do nothing. If you do nothing this pull request will be closed eventually by the stale bot. Please see CONTRIBUTING.md for more policy details."
+        stale-pr-label: "Stale"
+        exempt-pr-labels: "Keep" # a "Keep" label will keep the PR from being closed as stale
```

Comment:
Just want to double check this is something we want to support, letting users mark with a "keep" label to bypass the stale bot. Looks like it's optional.

Reply:
great question! my reasoning for including this was to allow for the option to keep a PR open for longer without needing to consistently ping the issue to keep it open. users can add a label if they're waiting for additional feedback from within their org or elsewhere, which has happened enough times that I elected to keep this option. if we see this being abused or overused, it's easy enough for us to remove the option to bypass the bot with this label entirely so we'll do that if needed